.config/tsaoptions.json

```diff
@@ -0,0 +1,12 @@
+{
+  "areaPath": "DevDiv\\ASP.NET Core",
```

Comment:
@BrennanConroy @halter73 what is the correct area for this repo❔

Reply:
This is the main outstanding question before we merge. Would also appreciate your 👀 on this 😀

Reply:
Ask a manager, no idea.

Reply:
Alright @rafikiassumani-msft or @danmoseley, what area path do you suggest for legacy SignalR❔

Reply:
DevDiv\ASP.NET OOB\SignalR

Reply:
```suggestion
  "areaPath": "DevDiv\\ASP.NET OOB\\SignalR",
```